driver/fvGFS/atmosphere.F90

```diff
@@ -611,33 +643,33 @@ subroutine atmosphere_dynamics ( Time )
       p_step = psc
                     call timing_on('fv_dynamics')
 !uc/vc only need be same on coarse grid? However BCs do need to be the same
-      call fv_dynamics(npx, npy, npz, nq, Atm(n)%ng, dt_atmos/real(abs(p_split)),&
-                       Atm(n)%flagstruct%consv_te, Atm(n)%flagstruct%fill,       &
-                       Atm(n)%flagstruct%reproduce_sum, kappa, cp_air, zvir,     &
-                       Atm(n)%ptop, Atm(n)%ks, nq,                               &
-                       n_split_loc, Atm(n)%flagstruct%q_split,                   &
-!                      Atm(n)%flagstruct%n_split, Atm(n)%flagstruct%q_split,     &
-                       Atm(n)%u,    Atm(n)%v,     Atm(n)%w,  Atm(n)%delz,        &
-                       Atm(n)%flagstruct%hydrostatic,                            &
-                       Atm(n)%pt  , Atm(n)%delp,  Atm(n)%q,  Atm(n)%ps,          &
-                       Atm(n)%pe,   Atm(n)%pk,    Atm(n)%peln,                   &
-                       Atm(n)%pkz,  Atm(n)%phis,  Atm(n)%q_con,                  &
-                       Atm(n)%omga, Atm(n)%ua,    Atm(n)%va, Atm(n)%uc,          &
-                       Atm(n)%vc,   Atm(n)%ak,    Atm(n)%bk, Atm(n)%mfx,         &
-                       Atm(n)%mfy , Atm(n)%cx,    Atm(n)%cy, Atm(n)%ze0,         &
-                       Atm(n)%flagstruct%hybrid_z,                               &
-                       Atm(n)%gridstruct,  Atm(n)%flagstruct,                    &
-                       Atm(n)%neststruct,  Atm(n)%idiag, Atm(n)%bd,              &
-                       Atm(n)%parent_grid, Atm(n)%domain,Atm(n)%diss_est)
-
-      call timing_off('fv_dynamics')
-
-      if (ngrids > 1 .and. (psc < p_split .or. p_split < 0)) then
+     call fv_dynamics(npx, npy, npz, nq, Atm(n)%ng, dt_atmos/real(abs(p_split)),&
+                      Atm(n)%flagstruct%consv_te, Atm(n)%flagstruct%fill,       &
+                      Atm(n)%flagstruct%reproduce_sum, kappa, cp_air, zvir,     &
+                      Atm(n)%ptop, Atm(n)%ks, nq,                               &
+                      n_split_loc, Atm(n)%flagstruct%q_split,                   &
+!                     Atm(n)%flagstruct%n_split, Atm(n)%flagstruct%q_split,     &
+                      Atm(n)%u,    Atm(n)%v,     Atm(n)%w,  Atm(n)%delz,        &
+                      Atm(n)%flagstruct%hydrostatic,                            &                       Atm(n)%pt  , Atm(n)%delp,  Atm(n)%q,  Atm(n)%ps,          &
```

Comment:
should be in two lines for pt, delp...?

Reply:
Thanks Jun - I've pushed a new commit for it.